java/bulk-export/bulk-export-core/src/main/java/sleeper/bulkexport/model/BulkExportLeafPartitionQueryJson.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022-2024 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.bulkexport.model;
+
+import java.util.List;
+
+/**
+ * A JSON representation of a request for a bulk export leaf partition query.
+ */
+class BulkExportLeafPartitionQueryJson {
```

Comment:
It doesn't look like this class is necessary, since it's identical to BulkExportLeafPartitionQuery?

Reply:
Class removed